### PR TITLE
updates

### PR DESCRIPTION
**Pull request by iruzevic**
updating fixes for active and focus state, helper for missing name and changing the locations of error selector

### CODE REVIEW

**Review by vedranpetrac — APPROVED** [review]

File: src/Blocks/components/checkboxes/components/checkboxes-options.js [hunk 1]
```diff
@@ -79,6 +79,7 @@ export const CheckboxesOptions = (attributes) => {
 				inlineLabel
 				noSearch
 				clearable
+				placeholder={__('Choose alternative', 'eightshift-forms')}
```
Comment (goranalkovic-infinum) [inline src/Blocks/components/checkboxes/components/checkboxes-options.js:82]:
```suggestion
				placeholder={__('Choose an alternative', 'eightshift-forms')}
```

File: src/Blocks/components/radios/components/radios-options.js [hunk 2]
```diff
@@ -60,6 +60,7 @@ export const RadiosOptions = (attributes) => {
 				inlineLabel
 				noSearch
 				clearable
+				placeholder={__('Choose alternative', 'eightshift-forms')}
```
Comment (goranalkovic-infinum) [inline src/Blocks/components/radios/components/radios-options.js:63]:
```suggestion
				placeholder={__('Choose an alternative', 'eightshift-forms')}
```

File: src/Blocks/components/select/components/select-options.js [hunk 3]
```diff
@@ -73,13 +73,13 @@ export const SelectOptions = (attributes) => {
 				inlineLabel
 				noSearch
 				clearable
+				placeholder={__('Choose alternative', 'eightshift-forms')}
```
Comment (goranalkovic-infinum) [inline src/Blocks/components/select/components/select-options.js:76]:
```suggestion
				placeholder={__('Choose an alternative', 'eightshift-forms')}
```

File: src/Blocks/components/utils/index.js [hunk 4]
```diff
@@ -332,14 +332,21 @@ export const getSettingsJsonOptions = (options, useEmpty = false) => {
...
 	if (value || asPlaceholder) {
 		return null;
 	}
 
+	const style = !isOptional ? 'es-bg-red-500' : 'es-bg-yellow-500';
+
 	return (
-		<div className={`es-position-absolute es-right-0 es-top-0 es-nested-color-pure-white es-bg-red-500 es-nested-w-5 es-nested-h-5 es-w-8 es-h-8 es-rounded-full es-has-enhanced-contrast-icon es-display-flex es-items-center es-content-center ${className}`}>
-			<Tooltip text={__('Name not set!', 'eightshift-forms')}>
+		<div className={`es-position-absolute es-right-0 es-top-0 es-nested-color-pure-white ${style} es-nested-w-5 es-nested-h-5 es-w-8 es-h-8 es-rounded-full es-has-enhanced-contrast-icon es-display-flex es-items-center es-content-center ${className}`}>
```
Comment (goranalkovic-infinum) [inline src/Blocks/components/utils/index.js:348]: Please use `classnames` helper from Libs

**Review by goranalkovic-infinum — CHANGES_REQUESTED** [review]

**Review by iobrado — APPROVED** [review]

**Review by dadadavorin — APPROVED** [review]

File: src/Blocks/components/utils/index.js [hunk 5]
```diff
@@ -332,14 +333,25 @@ export const getSettingsJsonOptions = (options, useEmpty = false) => {
...
+}) => {
 	if (value || asPlaceholder) {
 		return null;
 	}
 
+	const style = classnames([
+		'es-position-absolute es-right-0 es-top-0 es-nested-color-pure-white es-nested-w-5 es-nested-h-5 es-w-8 es-h-8 es-rounded-full es-has-enhanced-contrast-icon es-display-flex es-items-center es-content-center',
+		!isOptional ? 'es-bg-red-500' : 'es-bg-yellow-500',
+		selector(className, className),
+	]);
```
Comment (goranalkovic-infinum) [inline src/Blocks/components/utils/index.js:350]:
```suggestion
	const style = classnames(
		'es-position-absolute es-right-0 es-top-0 es-nested-color-pure-white es-nested-w-5 es-nested-h-5 es-w-8 es-h-8 es-rounded-full es-has-enhanced-contrast-icon es-display-flex es-items-center es-content-center',
		isOptional ? 'es-bg-yellow-500' : 'es-bg-red-500',
		className,
	);
```
The function handles most of the stuff for ya 😉

**Review by goranalkovic-infinum — CHANGES_REQUESTED** [review]

**Review by goranalkovic-infinum — APPROVED** [review]

**Pull request merged**